src/test/java/edu/harvard/iq/dataverse/api/UtilIT.java

```diff
@@ -2161,20 +2161,28 @@ static Response uploadProvFreeForm(String idOrPersistentId, JsonObject jsonObjec
 //        return requestSpecification.delete("/api/files/" + idInPath + "/prov-freeform" + optionalQueryParam);
 //    }
     static Response exportDataset(String datasetPersistentId, String exporter) {
-        return exportDataset(datasetPersistentId, exporter, null);
+        return exportDataset(datasetPersistentId, exporter, null, false);
     }
-
     static Response exportDataset(String datasetPersistentId, String exporter, String apiToken) {
+        return exportDataset(datasetPersistentId, exporter, apiToken, false);
+    }
+    static Response exportDataset(String datasetPersistentId, String exporter, String apiToken, boolean wait) {
 //        http://localhost:8080/api/datasets/export?exporter=dataverse_json&persistentId=doi%3A10.5072/FK2/W6WIMQ
         RequestSpecification requestSpecification = given();
         if (apiToken != null) {
             requestSpecification = given()
                     .header(UtilIT.API_TOKEN_HTTP_HEADER, apiToken);
         }
-        return requestSpecification
+        Response resp = requestSpecification
```

Comment:
This only needs to be called once, after waiting if wait=true.

Reply:
fixed